pdc-client.spec

```diff
@@ -147,7 +147,7 @@ cat > %{buildroot}/%{_sysconfdir}/pdc.d/fedora.json << EOF
     "fedora": {
         "host": "https://pdc.fedoraproject.org/rest_api/v1/",
         "develop": false,
-        "insecure": false
+        "ssl-verify": true
```

Comment:
@bliuredhat redhat's configuration should also be changed that way, right? It is not related to this patch, but it looks if this package is upgraded, the configuration above will overwrite the existing one. I am not sure if this part should be put in %post part to judge install or upgrade, and in upgrading just change this one item and keep others.